src/features/hotspots/setup/HotspotSetupBluetoothSuccess.tsx

```diff
@@ -1,66 +1,75 @@
 import { useNavigation } from '@react-navigation/native'
-import React, { useState } from 'react'
+import React from 'react'
 import { useTranslation } from 'react-i18next'
 import { Device } from 'react-native-ble-plx'
 import Box from '../../../components/Box'
-import Button from '../../../components/Button'
+import HotspotPairingList from '../../../components/HotspotPairingList'
 import Text from '../../../components/Text'
 import { useConnectedHotspotContext } from '../../../providers/ConnectedHotspotProvider'
 import { HotspotSetupNavigationProp } from './hotspotSetupTypes'
+import Bluetooth from '../../../assets/images/bluetooth.svg'
 
 const HotspotSetupBluetoothSuccess = () => {
   const { t } = useTranslation()
-  const [connecting, setConnecting] = useState(false)
   const navigation = useNavigation<HotspotSetupNavigationProp>()
-  const {
-    availableHotspots,
-    connectAndConfigHotspot,
-  } = useConnectedHotspotContext()
+  const { availableHotspots } = useConnectedHotspotContext()
```

Comment:
you can mock the following for testing to see all the available hotspot types:

```js
 const availableHotspots = {
    '31D15CD5': {
      id: '31D15CD5',
      localName: 'Helium Hotspot A15B',
      name: 'Helium Hotspot',
    },
    '61D15CD5': {
      id: '61D15CD5',
      localName: 'Rak Hotspot Miner F7B4',
      name: 'Helium Hotspot',
    },
    '41D15CD5': {
      id: '41D15CD5',
      localName: 'Nebra Indoor Hotspot A15BFF',
      name: 'Helium Hotspot',
    },
    '51D15CD5': {
      id: '51D15CD5',
      localName: 'Nebra Outdoor Hotspot A15BCC',
      name: 'Helium Hotspot',
    },
  }
```

Reply:
Do we also need to add these to `HotspotSelectionScreen.tsx`?

Reply:
we probably will, but don't have designs or copy for this yet. @cokes518 has some WIP docs for how makers can define their assets and onboarding instructions